packages/hierarchies/src/hierarchies/internal/operators/Grouping.ts

```diff
@@ -33,18 +34,18 @@ export function createGroupingOperator(
       log({ category: LOGGING_NAMESPACE, message: /* istanbul ignore next */ (n) => `in: ${createNodeIdentifierForLogging(n)}` }),
       toArray(),
       concatMap((resolvedNodes) => {
+        const { instanceNodes, restNodes } = partitionInstanceNodes(resolvedNodes);
```

Comment:
Could we pass `instanceNodes` to `createGroupingHandlers` to avoid the need to filter `resolvedNodes` there again?

Reply:
That's right. Done